packages/ts-logger/src/logger.spec.ts

```diff
@@ -16,17 +16,14 @@ describe('createLogger', () => {
     jest.resetAllMocks();
   });
 
-  it('Local environment writes to files only', async () => {
-    process.env.NODE_ENV = 'local';
+  it('Local environment does not write to file', async () => {
```

Comment:
I needed logs locally during development in the console. I think this is fine to enable as long as they are not chatty.